core/rs/core/src/c.rs

```diff
@@ -281,7 +282,7 @@ fn bindgen_test_layout_crsql_ExtData() {
         )
     );
     assert_eq!(
-        unsafe { ::core::ptr::addr_of!((*ptr).pPragmaDataVersionStmt) as usize - ptr as usize },
+        unsafe { ::std::ptr::addr_of!((*ptr).pPragmaDataVersionStmt) as usize - ptr as usize },
```

Comment:
these should all be `core`

`bindgen --use-core crsqlite.h` will switch this from `std` to `core`